routers/api/v1/admin/hooks.go

```diff
@@ -36,7 +36,7 @@ func ListHooks(ctx *context.APIContext) {
 	//   "200":
 	//     "$ref": "#/responses/HookList"
 
-	sysHooks, err := webhook.GetSystemWebhooks(ctx, util.OptionalBoolNone)
+	sysHooks, err := webhook.GetSystemOrDefaultWebhooks(ctx, util.OptionalBoolNone)
```

Comment:
I don't think we should change the behaviour? Maybe we should have parameter or we could have another endpoint for default webhooks?